scout/server/blueprints/cases/controllers.py

```diff
@@ -655,6 +655,7 @@ def phenotypes_genes(store, case_obj):
             "genes": ", ".join(sorted(gene_list)),
         }
 
+    # Case where dynamic_panel_phenotypes is empty, perhaps because user has added custom genes to HPO panel
```

Comment:
A set catching extras may still have been a more elegant way to do this, but yeah, I do think this covers it. 💯

Reply:
Thanks for the review!